### ISSUE TITLE
Obey global excludes when importing

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zaproxy#8022]
For all import options from URLs:

- [ ] Exim
- [ ] GraphQL
- [ ] OpenAPI
- [ ] Postman
- [ ] SOAP

Ideally support would be added in network and the above add-ons would change to warn / notify the user that specific endpoints were ignored.

**Comment by kingthorin** [comment]
This probably needs to be more than just Global? Don't most of them use spider?

**Comment by kingthorin** [comment]
Some may also need to check mode? (That's just a thought, I haven't checked code.)

**Comment by thc202** [comment]
IMO it's not responsibility of network add-on to decide whether a request should be accessed or not.

**Comment by psiinon** [comment]
Agreed - the network add-on is too low level to make decisions like that.
